examples/experimental/dagster-airlift/dagster_airlift/core/sensor/event_translation.py

```diff
@@ -13,6 +13,10 @@
 from dagster_airlift.core.airflow_defs_data import AirflowDefinitionsData
 from dagster_airlift.core.airflow_instance import DagRun, TaskInstance
 
+AirflowEventTranslationFn = Callable[
+    [DagRun, Sequence[TaskInstance], AirflowDefinitionsData], List[AssetMaterialization]
```

Comment:
Might as well be Iterable[AssetMaterialization] so people can build a function that yields

Reply:
works for me